src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Helper/TranslationHelper.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Microsoft.PowerToys.Run.Plugin.WindowsSettings.Properties;
+using Wox.Plugin.Logger;
+
+namespace Microsoft.PowerToys.Run.Plugin.WindowsSettings.Helper
+{
+    /// <summary>
+    /// Helper class to easier work with translations.
+    /// </summary>
+    internal static class TranslationHelper
+    {
+        /// <summary>
+        /// Translate all settings of the given list with <see cref="WindowsSetting"/>.
+        /// </summary>
+        /// <param name="settingsList">The list that contains <see cref="WindowsSetting"/> to translate.</param>
+        internal static void TranslateAllSettings(in IEnumerable<WindowsSetting>? settingsList)
+        {
+            if (settingsList is null)
+            {
+                return;
+            }
+
+            foreach (var settings in settingsList)
+            {
+                var area = Resources.ResourceManager.GetString($"Area{settings.Area}");
```

Comment:
Let's use:

```suggestion
                var area = Resources.ResourceManager.GetString($"Area.{settings.Area}");
```

=> JSON NEEDS TO BE UPDATED THEN.


This will fix a bug where with an empty/missing area setting the commamd returns the resource string for `area` and translation become false.

**At the moment:**
The if statement in line 35 returns always true becaus the string can be translated. And in line 50 the existing value is kept. This leads e.g. to the line `Area: Area` in the tool tip.

Reply:
Additional Log should be improved to 
```c#
  Log.Warn($"Resource string for [Area.{settings.Area}] not found or area is not defined", typeof(Main));
```

Reply:
Note: We can fix it after merging in a second PR/step too.

Reply:
We we use a point `.` in the resources file it is translated to a underscore `_`, so we must used a underscore `_` in the code and not a point `.` .

When you are okay with this I will change it.

Reply:
Lets merge as is. And I will create a tracking issue with all suggesti improvements.